samples/DatabaseProjectAFSqlExtensions/DatabaseProjectAFSqlExtensions.sqlproj

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Have you considered using the newer and simpler SDK-style .sqlproj format supported in Azure Data Studio?  It'd really clean this file up!

Reply:
Others have mentioned it.  I had not seen it as an option until today.   I'll be doing it tomorrow.